src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -276,3 +276,19 @@ export const retireConcept = (conceptUrl, retired) => async (dispatch) => {
     return null;
   }
 }
+
+export const addReferenceToCollectionAction = (type, owner, collection, expressions) => async (dispatch) => {
+  try {
+    return await api.dictionaries.addReferencesToCollection(type, owner, collection, expressions);
+  } catch (e) {
+    return false;
```

Comment:
Isn't the caller interested in knowing the cause of the failure?

Reply:
Oversight. Thanks for pointing this out.